### PR TITLE
Make local database optional.

### PR DESCRIPTION
**Pull request by davidlehn**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,29 +1,37 @@
...
-## [Unreleased]
+## Unreleased
 
-## [2.1.4] - 2016-05-23
+### Changed
+- Add `config.mongodb.local.enable` option to enable using a local database.
+  Disabled by default. In a remote replicated database setup, the local
+  database needs to actually be local. Running a full mongodb node is
+  heavyweight for the purposes of this module. For the time being, if the local
+  database is disabled, every startup of a server will get a new globalId.  A
```
Comment (dlongley) [inline CHANGELOG.md]: Every new process will get a new globalId. (not every server -- the definition of that is ambiguous)

File: lib/idGenerator.js [hunk 2]
```diff
@@ -135,6 +135,9 @@ function _loadIds(idGenerator, callback) {
       var fields = {};
       fields[globalKey] = true;
       fields[localKey] = true;
+      if(!database.localCollection) {
```
Comment (dlongley) [inline lib/idGenerator.js]: This should happen above the query, no reason to run the other lines above it.

**Comment by dlongley** [comment]
Just a couple comments to address then LGTM.

**Comment by davidlehn** [comment]
Comitted in https://github.com/digitalbazaar/bedrock-mongodb/commit/f607ea7ea1347eee82397e80d16c7bf658211036

**Pull request closed without merging**